Source/Filter/NCIGodfreyFilter.cpp

```diff
@@ -23,11 +23,12 @@
 using namespace amrex;
 
 NCIGodfreyFilter::NCIGodfreyFilter(godfrey_coeff_set coeff_set, amrex::Real cdtodz, bool nodal_gather){
-#if (AMREX_SPACEDIM >= 2)
     // Store parameters into class data members
     m_coeff_set = coeff_set;
     m_cdtodz = cdtodz;
     m_nodal_gather = nodal_gather;
+
+#if (AMREX_SPACEDIM >= 2)
```

Comment:
I think, Lines 31-43 can be simplified with 
`#if (AMREX_SPACEDIM == 3)
    stencil_length_each_dir = {1,1,5};
    slen = {1,1,5};
#elif (AMREX_SPACEDIM == 2)
    stencil_length_each_dir = {1,5};
    slen = {1,5,1};
#else
    amrex::ignore_unused(coeff_set, cdtodz, nodal_gather);
    amrex::Abort("NCIGodfreyFilter not implemented in 1D!");
#endif`

Reply:
```suggestion
```

Reply:
yep, haha what did I do here :D